### PR TITLE
[Service] util function to request raw data

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add util function to request offloading service - send raw data.

### CODE REVIEW

File: c/src/ml-api-service-offloading.c [hunk 1]
```diff
@@ -1184,3 +1184,20 @@ ml_service_offloading_request (ml_service_h handle, const char *key,
     nns_edge_data_destroy (data_h);
   return ret;
 }
+
+/**
+ * @brief Register new information, such as neural network models or pipeline descriptions, on a offloading server.
```
Comment (songgot) [inline c/src/ml-api-service-offloading.c:1189]: The contents of the brief are different from ml-api-service-offloading.h line 103.
Did you intend to?

File: c/src/ml-api-service-offloading.c [hunk 2]
```diff
@@ -1184,3 +1184,20 @@ ml_service_offloading_request (ml_service_h handle, const char *key,
...
+int
+ml_service_offloading_request_raw (ml_service_h handle, const char *key,
+    void *data, size_t len)
+{
+  ml_tensors_data_s input;
+
+  /* Set internal data structure to send edge data. */
+  input.num_tensors = 1;
+  input.tensors[0].data = data;
+  input.tensors[0].size = len;
```
Comment (songgot) [inline c/src/ml-api-service-offloading.c:1200]: Is there no need to set dimension and type?

Comment (jaeyun-jung) [reply c/src/ml-api-service-offloading.c:1200]: nns-edge does not check tensor shape or type. It handles data only.

File: c/src/ml-api-service-offloading.c (hunk 1 above)
Comment (jaeyun-jung) [reply c/src/ml-api-service-offloading.c]: I added comment, same contents w/ml_service_offloading_request() function.

File: c/src/ml-api-service-offloading.c (hunk 2 above)
Comment (songgot) [reply c/src/ml-api-service-offloading.c:1202]: Thank you.

**Review by songgot — APPROVED** [review]
LGTM

**Pull request merged**